beets/dbcore/query.py

```diff
@@ -138,6 +138,22 @@ def value_match(cls, pattern, value):
         return pattern == value
 
 
+class EmptyQuery(FieldQuery):
+    """A query that looks for an empty item field."""
+    def col_clause(self):
+        return self.field + " = ?", [self.pattern]
```

Comment:
To work the same way as match, we should probably use "field IS NULL OR field = '' OR field = 0` or somesuch. Or, if SQLite supports it, then perhaps just `NOT field` will work?